app/user/rest.py

```diff
@@ -66,9 +69,20 @@ def update_user(user_id):
     return jsonify(data=user_schema.dump(user_to_update).data), 200
 
 
+@user.route('/<uuid:user_id>/update-attribute', methods=['PUT'])
```

Comment:
Should this endpoint be:

POST /user/<id>

That would allow updating of fields without the need for a new path.